internal/cmd/cmd.go

```diff
@@ -294,6 +295,10 @@ func newConnectorCmd() *cobra.Command {
 				return err
 			}
 
+			// Also accept the same env var as the CLI for setting the access key
+			if accessKey, ok := os.LookupEnv("INFRA_ACCESS_KEY"); ok {
+				options.Server.AccessKey = types.StringOrFile(accessKey)
```

Comment:
If the access key is passed through environment variables, it should contain the actual access key and not a file reference. `StringOrFile` is unnecessary

Reply:
Ya, in this case it is unnecessary, but it has to be used here because that's the type of the `AccessKey` field.

The field is that type because in other scenarios we do load it from a file.

Reply:
Change this to use `Set` on the StringOrFile value, so that we can specify the name of a file in the env var as well. As we discussed we'll want this for case 3 in https://github.com/infrahq/infra/pull/3220#discussion_r975843443